invadedhouse/commands/pickup.py

```diff
@@ -0,0 +1,30 @@
+from commands import Command
+
+
+class Pickup(Command):
+
+    def do(self, text: str):
+        player = self.get_player()
+        current_room = player.get_location()
+        item_str = text
+
```

Comment:
You could add "all" here kinda like what I did in Drop, just reverse.

Reply:
no a want to do this with loot; i want to do 'loot floor'
thats my idea

Reply:
yep you can do that too! DO IT! 😁